azavea/raster-foundry
# Clean up leaflet-side-by-side

## Overview

Clean up leaflet-side-by-side
* Fix bug where dragging slider on chrome>55 causes map to no longer respond to
clicks
* Move css to separate file and format
* Use es6 notation so eslint can catch stuff

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~

### Notes
This still needs quite a bit of work to look nice, but at least it's readable / styleable now
without a bunch of commonjs code.

There's apparently a way to extend leaflet using es6 classes (at least one
person has tweeted about it) but I haven't found an actual code sample /
explanation on how to do it.

## Testing Instructions

 * Select a Tool, and run it on two projects which have been ingested
 * Compare two nodes
 * Verify that the split screen acts as expected
 * On a chrome version > 55, verify that moving the screen split no longer breaks map interaction

Fixes #987
